### PR TITLE
S4/75 76 77 78 79 security service partie ma demande

### PR DESCRIPTION
**Pull request by IbtissameArrai**

### CODE REVIEW

File: notification_service/src/main/java/com/eai/notification_service/controller/SecurityServiceEmailController.java [hunk 1]
```diff
@@ -26,4 +26,26 @@ public String sendOtpEmail(@RequestBody EmailSender request) {
 
        return emailSenderService.sendOtpEmail(request.getEmail(), subject, variables, templatePath);
    }
+
+    @PostMapping("/send-exist-email")
```
Comment (souhaib-eai) [inline notification_service/src/main/java/com/eai/notification_service/controller/SecurityServiceEmailController.java:30]: put this code in a service

File: notification_service/src/main/java/com/eai/notification_service/controller/SecurityServiceEmailController.java [hunk 2]
```diff
@@ -26,4 +26,26 @@ public String sendOtpEmail(@RequestBody EmailSender request) {
...
+        variables.put("codeOtpEmail", request.getCodeOtpEmail());
+        variables.put("subject", subject);
+
+        return emailSenderService.sendOtpEmail(request.getEmail(), subject, variables, templatePath);
+    }
+
+    @PostMapping("/send-email-login")
+    public String sendEmailRegister(@RequestBody EmailSender request){
+        String subject = "Agence directe : Avez-vous de la difficulté à ouvrir une session dans votre compte?";
+        String templatePath = "otpService/email/send-login-template.html";
```
Comment (souhaib-eai) [inline notification_service/src/main/java/com/eai/notification_service/controller/SecurityServiceEmailController.java:45]: put this code in a service

**Review by souhaib-eai — CHANGES_REQUESTED** [review]

File: notification_service/src/main/java/com/eai/notification_service/controller/SecurityServiceEmailController.java (hunk 2 above)
Comment (IbtissameArrai) [reply notification_service/src/main/java/com/eai/notification_service/controller/SecurityServiceEmailController.java]: done

File: notification_service/src/main/java/com/eai/notification_service/controller/SecurityServiceEmailController.java (hunk 1 above)
Comment (IbtissameArrai) [reply notification_service/src/main/java/com/eai/notification_service/controller/SecurityServiceEmailController.java:33]: done

**Pull request closed without merging**